tests/providers/google/cloud/operators/test_dataproc_system.py

```diff
@@ -54,12 +75,14 @@ def setUp(self):
         self.create_gcs_bucket(BUCKET)
         self.upload_content_to_gcs(lines=pyspark_file, bucket=GCS_URI, filename=PYSPARK_MAIN)
         self.upload_content_to_gcs(lines=sparkr_file, bucket=GCS_URI, filename=SPARKR_MAIN)
+        self.upload_content_to_gcs(lines=yml_config_file, bucket=GCS_URI, filename=YML_CONFIG_FILE)
```

Comment:
Using both `yml_config_file` and `YML_CONFIG_FILE` is bad for readability in my opinion, maybe one should be renamed to reduce any confusion.